ddp-workspace/projects/ddp-pancan/src/styles/common.scss

```diff
@@ -56,6 +58,49 @@
     letter-spacing: -0.016rem;
 }
 
+.center {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+}
+
+// for Welcome page / Colorectal page mostly
+section {
```

Comment:
I'm getting rid of this file in https://github.com/broadinstitute/ddp-angular/pull/759, so if it's possible let's move to the styles.scss or create a separate file
also I can do it in me PR after this one will be merged

Reply:
Done